client/api/users/service.ts

```diff
@@ -22,44 +22,25 @@ const updateUsersLanguages = async (
   }
 };
 
-const registerUser = async (
-  firstName: string,
-  lastName: string,
-  email: string,
-  password: string,
-  roleId: number,
-  languageId: number,
-  groupId: number | null, // they might be joining an existing group
-  name: string | null,
-  streetAddress: string | null,
-  city: string | null,
-  state: string | null,
-  zipCode: string | null,
-  // logoUrl: string | null, // still need to set up s3 buckets for images so for now we wont collect this and just set it to null
-) => {
+const registerUser = async (data: RegisterUser) => {
   try {
-    const registerData = {
-      first_name: firstName,
-      last_name: lastName,
-      email: email,
-      password: password,
-      role_id: roleId ? roleId : 1,
-      language_id: 1,
-      group_id: groupId,
-      name: name,
-      street_address: streetAddress,
-      city: city,
-      state: state,
-      zip_code: zipCode,
-      // logo_url: logoUrl,
+    const formData = {
+      group_id: data.group_id,
+      name: data.name,
+      streetAddress: data.streetAddress,
+      city: data.city,
+      state: data.state,
+      zipCode: data.zipCode,
+      logoUrl: data.logoUrl,
```

Comment:
I believe we can [spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) here fort his object

```suggestion
      ...data
```

Reply:
Perfect use of spread actually! Thank you!